packages/code-du-travail-frontend/src/outils/publicodes/indemnite-licenciement.ts

```diff
@@ -14,6 +15,7 @@ export const mapToPublicodesSituationForCalculation = (
     "contrat salarié . indemnité de licenciement . date de notification":
       notificationDate,
     "contrat salarié . indemnité de licenciement . date de sortie": endDate,
+    dateArret: stopDate,
```

Comment:
pour moi, il faut qu'on fasse du style : 

```suggestion
   "contrat salarié . indemnité de licenciement . date d'arret de travail": stopDate,
```

même si on s'en sert pas que côté publicodes, ça permet de garder une cohérence

Reply:
Si on renomme en contrat salarié . indemnité de licenciement . date d'arret de travail dans ce cas il faut aussi que j'initialise la variable dans le yaml parce que le calculate a un système de filtre sur le variable. Si elle commence par "contrat salarié . " alors elle le transmet dans le publicode
Hors je n'ai pas besoin de l'envoyer dans le yaml donc je ne pense pas que ce soit une bonne idée

Reply:
Pour moi, c'est pas déconnant de leur rajouter dans le yaml, vu que c'est du métier

Reply:
car sinon, ça serait la seule variable qui aurait une logique différente